css/index.css

```diff
@@ -137,10 +182,109 @@ footer nav {
     justify-content: space-between;
     align-items: center;
     padding: 20px 2%;
-    font-size: 14px;
+    font-size: .8rem;
 }
 
 footer nav a {
     color: white;
     text-decoration: none;
+}
+
+/* Ipad */
+@media (max-width: 768px) {
+   
+    header {
+        display: block;
```

Comment:
You could have left `header` as a flex container with `flex-direction: column` and `align-items: center`. That way you don't have to use hacky tricks like `margin-left: 38.5%` below.